apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -122,3 +130,35 @@ $tokensTable-border-width: 1px;
         border-width: 0;
     }
 }
+
+.c-tokensTable-sectionHeading,
+.c-tokensTable-sectionDescription,
+.c-tokensTable-sectionSubheading {
+    max-width: v.$content-max-width;
+}
+
+.c-tokensTable-sectionSubheading {
+    margin-top: f.spacing(f);
+
+    &:not(:first-of-type) {
+        margin-top: f.spacing(i);
+        position: relative;
+        &:before {
+            content: '';
+            position: absolute;
+            left: 0;
+            right: 0;
+            top: -#{f.spacing(f)};
+            background-color: f.$color-border-default;
+            height: $tokensTable-border-width;
```

Comment:
Is this to apply a border? If so, would be useful to have a comment before the `&:before` just saying that (or what it is styling)